base/commands/project/utils.go

```diff
@@ -31,22 +31,13 @@ func loadFromDefaults(templateDir string) (map[string]string, error) {
 	if err = parseYAML("", b, props); err != nil {
 		return nil, err
 	}
-	props = camelizeMapKeys(props)
-	if err != nil {
-		return nil, err
+	if key, ok := validateMap(props); !ok {
```

Comment:
`validateMap` is so generic. I would suggest something like `validateKeyFormat`, `validatePropKeys` etc.